test/halmos.toml

```diff
@@ -2,7 +2,6 @@
 # Test settings.
 function = "testHalmos"       # Run tests matching the given prefix.
 storage-layout = "generic"    # Set the generic storage layout model.
-test-parallel = true          # Run tests in parallel.
```

Comment:
What's the underlying reason why this flag is a net negative for performance?

Reply:
there are trade-offs:
- setup() is not saved across multiple tests, but re-executed for each test when test-parallel enabled.
- if both test-parallel and solver-parallel are enabled, the (peak) number of parallel solver processes might be too high, potentially degrading performance.